man/src/network/create.md

```diff
@@ -134,3 +134,44 @@ $ docker network create -d overlay \
   --opt encrypted=true \
   my-ingress-network
 ```
+
+### Run services on predefined networks
+
+You can now create services on the predefined docker networks `bridge` and `host`.
+
+```bash
+$ docker service create --name my-service \
+  --network host \
+  --replicas 2 \
+  busybox top
+```
+
+### Swarm networks with local scope drivers
+
+You can now create a swarm network with local scope network drivers. You do so
+by promoting the network scope to `swarm` during the creation of the network. 
+You will then be able to use this network when creating services. 
+
+```bash
+$ docker network create -d bridge \
+  --scope swarm \
+  --attachable \
+  swarm-network
+```
+
+For network drivers which provide connectivity across hosts (ex. macvlan), if
+node specific configurations are needed in orther to plumb the network on each
+host, you will supply that configuration via a configuration only network.
+When you create the swarm scoped network, you will then specify the name of the 
+network which contains the configuration.
+
+
+```bash
+node1$ docker network create --config-only --subnet 192.168.100.0/24 --gateway 192.168.100.115 mv-config
```

Comment:
Would there be practical examples where a `--config` is reusable for multiple networks?


Mixing "configs" ("templates") with actual networks in the same list is confusing (how do I identify a network as "config-only"?);

```bash
$ docker network create --config-only --scope=swarm swarm-config
$ docker network create --config-only --driver=bridge bridge-config

$ docker network ls
NETWORK ID          NAME                DRIVER              SCOPE
f465dd713f62        bridge              bridge              local
4b40b17e4a72        bridge-config       null                local
f0c63ab67493        docker_gwbridge     bridge              local
be0f334e7171        host                host                local
q1lmvx3qs5gb        ingress             overlay             swarm
9a73bd22e688        none                null                local
314332c82133        swarm-config        null                local
```


The errors are confusing as well when trying to use such a network;

```bash
$ docker service create --name foo --network=mv-config nginx:alpine
Error response from daemon: The network mv-config cannot be used with services.
Only networks scoped to the swarm can be used, such as those created with the overlay driver.
```

Slightly better with `docker run`

```bash
$ docker run -dit --name foo --network=mv-config busybox
480102cb5a3289539c26148dcd4ad931175cb62b9b3a7759a7ac5674b375ab1c
docker: Error response from daemon: cannot create endpoint on configuration-only network.
```